crates/biome_js_analyze/tests/specs/nursery/useSingleJsDocAsterisk/valid.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * **bold**
+ */
+
+/**
+ * ** *** ** Not checked after double asterisk
+ */
+
+/**
+ * Valid end, single asterisk at the start */
+
+/**
+ ** * Valid end, double asterisk at the start */
+
+/**
+ Asterisk after text * *
+*/
```

Comment:
I feel this one should be invalid, no?

Reply:
https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/no-multi-asterisks.md Allows it.

I am reaching a bit but could imagine comments like:
```
/**
* https//google.com/* 
* Password ****
*/
```
Unintentional asterisks at the end of line (except last one) should be rare enough, so for now I would it leave as is, but no strong preference tho.

Reply:
I don’t necessarily mind the asterisk at the end, but shouldn’t it fail because there isn’t an asterisk at the start of the line?

Reply:
I copied the same behavior from [no-multi-asterisks](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/no-multi-asterisks.md) and did not think much of it, but seems like oversight.

We should catch it, though I would like to implement this in a separate PR not to stall this one.